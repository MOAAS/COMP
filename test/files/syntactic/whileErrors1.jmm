class Worker extends Person {
    int idade;
    boolean is_working123;
    int[] favorite_$_Numbers;
    Identification id;

    public static void main(String[] args) {
        int x;

        while (a[4 < 4) {

        }

        while (true {

        }

        while() {

        }

        while true) {
            while (true && ) {

            }

            while ( < 4 ) {

            }
        }
    }
}