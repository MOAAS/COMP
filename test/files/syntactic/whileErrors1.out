Parse Error found on while expression: line 10, column 23.
Error Details:
	The expression is missing a closing bracket ']'

Parse Error found on while expression: line 14, column 21.
Error Details:
	The expression is missing a ')' at the end.

Parse Error found on while expression: line 18, column 15.
Error Details:
	The expression cannot be empty.

Parse Error found on while expression: line 22, column 15.
Error Details:
	The expression is missing a '(' after the while keyword.

Parse Error found on while expression: line 23, column 28.
Error Details:
	The expression is missing an argument after the operator "&&"

Parse Error found on while expression: line 27, column 21.
Error Details:
	The expression is missing an argument before the operator "<"

